tests/scripts/analyze_outcomes.py

```diff
@@ -211,6 +210,44 @@ def do_analyze_driver_vs_reference(outcome_file, args):
                 'test_suite_random': [
                     'PSA classic wrapper: ECDSA signature (SECP256R1)',
                 ],
+                # In the accelerated test ECP_C is not set (only ECP_LIGHT is)
```

Comment:
Why can't we also only enable ECP_LIGHT in component_test_psa_crypto_config_reference_all_ec_algs_use_psa instead?

Reply:
For the same reason we're not disabling `ECDSA_C` in the reference component: we want the feature set of ECC-using modules (PK, X.509, TLS) with `ECP_LIGHT && !ECP_C` to be as close as possible to the feature set with `ECP_C`. That's why we're keeping `ECP_C` enabled in the component, and have an ignore list for `test_suite_ecp`. This is similar to how we have `ECDSA_C` in the reference but ignore the whole `test_suite_ecdsa` - except here we don't ignore the whole `test_suite_ecp`, just part of it.